test/proxy_recover_funds.js

```diff
@@ -129,47 +130,26 @@ contract('Proxy funds', accounts => {
               const vaultProxyAddress = getEvent(receipt, 'NewAppProxy', 'proxy')
               vault = VaultMock.at(vaultProxyAddress)
             }
+            await vault.initialize()
+
             await kernel.setApp(APP_ADDR_NAMESPACE, vaultId, vault.address)
             await kernel.setRecoveryVaultAppId(vaultId)
           })
 
-          onlyBaseKernel(() => {
```

Comment:
`onlyBaseKernel` and `onlyKernelProxy` are not used anymore, right? Then we can remove above (lines 81, 82)